dodona-bash.dockerfile

```
@@ -4,6 +4,7 @@ RUN apt-get update && \
     apt-get -y install --no-install-recommends \
         bc=1.07.1-2+b1 \
         binutils=2.31.1-16 \
+        bsdmainutils=12.1.7 \
```

```suggestion
        bsdmainutils=11.1.2+b1 \
```
This is the version for debian buster (tested locally).

---

Yes, this should be the one :) Thanks!